components/tooltip/index.js

```diff
@@ -178,7 +178,7 @@ class Tooltip extends Component {
 					className="components-tooltip"
 					aria-hidden="true"
 				>
-					{ text }
+					{ upperFirst( toLower( text ) ) }
```

Comment:
couldn't this clash with different languages requirements?

Reply:
I did some tests with different languages like greek, chinese, korean, russian, persian and hindi. I did not found issues. But it was a quick test. Probably it's necessary investigate more.

Maybe it's better to try via an ESlint rule or equivalent as aduth said.